src/components/ViewRoutingSlip/RefundRequestForm.vue

```diff
@@ -1,25 +1,51 @@
 <template>
   <v-form ref="refundRequestForm">
-    <v-text-field
-    filled
-    label="Name of Person or Organization"
-    persistent-hint
-    v-model.trim="name"
-    data-test="txtName"
-    :rules="nameRules"
-    v-if="isEditing"
-    >
-    </v-text-field>
-    <span v-else>{{ name }}</span>
-    <address-form
-      ref="addressForm"
-      :editing="isEditing"
-      :schema="baseAddressSchema"
-      :address="address"
-      @update:address="address=$event"
-      @valid="addressValidity"
-    >
-    </address-form>
+    <v-row>
+      <v-col class="col-3 font-weight-bold pb-0">
+        {{ 'Name of Person or Organization & Address' }}
+      </v-col>
+      <v-col class="col-9 pb-0">
+        <v-text-field
+        filled
+        label="Name of Person or Organization"
+        persistent-hint
+        v-model.trim="name"
+        data-test="txtName"
+        :rules="nameRules"
+        v-if="isEditing"
+        >
+        </v-text-field>
+        <span v-else>{{ name }}</span>
+        <address-form
+          ref="addressForm"
+          :editing="isEditing"
+          :schema="baseAddressSchema"
+          :address="address"
+          @update:address="address=$event"
+          @valid="addressValidity"
+        >
+        </address-form>
+      </v-col>
+      <v-col class="col-3 font-weight-bold"
+        :class="isEditing ? 'pt-0' : ''">
+        Cheque Advice
+      </v-col>
+      <v-col
+        class="col-9"
+        :class="isEditing ? 'pt-0' : ''">
+        <v-text-field
+        filled
+        label="Additional Information"
```

Comment:
Is the label correct ?

Reply:
![image](https://user-images.githubusercontent.com/78374865/147314133-c58bbeec-11a6-4489-9ef3-d2c0054d22e8.png)